dashboard/src/components/AlertBenchUpdate.vue

```diff
@@ -110,14 +110,6 @@ export default {
 		}
 	},
 	computed: {
-		show() {
```

Comment:
removing this since the status of bench will always be one among Active and Awaiting Deploy so using `deployInformation?.update_available` instead